labs/docker-101/3.md

```diff
@@ -2,131 +2,126 @@
       ![Docker Images](assets/images/images1.png)
 
-5. __Run the new image__. Run the newly created docker image by giving the randomly-chosen ID to Docker:
+5. Run the newly created docker image by giving the randomly-chosen ID to Docker:
    ```
    docker run -ti <your image ID>
    ```  
-   The newly-created image behaves exactly the same way as the Ubuntu image from Dockerhub because was built from the same Dockerfile. You now have a bash root prompt. Enter ``ls`` to confirm standard Ubuntu directories are installed.
+   The newly-created image behaves exactly the same way as the Ubuntu image from Dockerhub, because it is built from the same Dockerfile. You now have a bash root prompt.
+   
+ 6. Enter ``ls`` to confirm standard Ubuntu directories are installed.
 
 # Building a Custom Cisco Learning Labs Container
 
-To modify the image that we just built, all we have to do is change the Dockerfile and ensure that any needed resources are available in the repository. Let's make an image that's more specific to our application's needs.
+To modify the image that we just built, all we have to do is change the Dockerfile and ensure that all required resources are available in the repository. Let's make an image that's more specific to our application's needs.
 
-I'm currently sitting in at Cisco LIVE! Europe 2017 in the wonderful
-city of Berlin, so let's change the image to say that when it runs:
+Let's change the image to display the following message:
 
-![Custom Docker Output](assets/images/hellocustom1.png)
+ ```
+ Hello from DevNet!
+ ```
 
-To make the image display that message we will:
+To make the image display the above message we will:
 
 1. Create a new Python script.  
 
 2. Edit the Dockerfile to include that script in the container build.  
 
-3. Change the Dockerfile to install python in the container (remember,
-all dependencies must be present in the container).  
+3. Change the Dockerfile to install python in the container (remember, all dependencies must be present in the container).  
 
 4. Build and test the new container.  
 
-You have a choice young Jedi:
 
-You can either
+To update the image, follow the below instructions:
 
-* Copy and paste following Dockerfile commands
-* Type them into the file manually
-* Download the Github repository that contains them
+1. Clone the Github repository that contains the Dockerfile for this section using the following command:
 
-To clone the Github repository that contains the Dockerfile for this
-section using the following command:
+     ```
+    git clone https://github.com/CiscoDevNet/container-intro-devnet.git
+    cd container-intro-devnet
+    ```
 
-```
-git clone https://github.com/CiscoDevNet/container-intro-devnet.git
-cd container-intro-devnet
-```
+   The cloned repository contains two files:
```

Comment:
It contains four files, so reword as: "The cloned repository contains these files:"